test-harness/src/main/java/io/jenkins/plugins/casc/misc/RoundTripAbstractTest.java

```diff
@@ -108,7 +112,7 @@ public void roundTripTest() throws IOException {
             assertConfigViaWebUI(jenkinsConf);
 
             //Validate Schema before application
-            assertThat(validateSchema(convertToJson(resourceContent)), empty());
+            assertThat(validateSchema(new JSONObject(new JSONTokener(convertToJson(resourceContent)))), empty());
```

Comment:
why is this needed?

Reply:
Because of the type expected by ValidateSchema.

Reply:
We can use the function directly but would need some modifications.

Reply:
probably worth looking at? this seems more complex than expected? maybe another helper method?

Reply:
Implemented